### DISCUSSION TITLE
Editor Focus and SetSelectionRange

### DISCUSSION THREAD

**Discussion by jflerime314** [sibiraj-s/ngx-editor#498] [Q&A] [answered]
1. I know there are props that exist for the focus event but I want to know how I can set focus from code if I add a new component of the editor to a list. 
2. 
3. I also want to know how to replicate the input setSelectionRange behavior with the editor. I want to move the cursor to the first position of the next editor input on a keydown event.

**Comment by sibiraj-s** [comment, marked as answer]
Please refer the list of commands here. https://sibiraj-s.github.io/ngx-editor/en/commands/. Not all commands are documented yet, but autocomplete in IDE should be able to giveout things for now.  Or can be referred here https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/src/lib/EditorCommands.ts

> I want to move the cursor to the first position of the next editor input on a keydown event.
not sure I understand it completely. can you elaborate?

**Reply by jflerime314** [reply]
Let's say I have a list of ngx-editor in a page.

section 1

section 2

section 3

While I'm in the section 1 editor, I want to press the arrow down move the section 2 editor, and place the cursor in the beginning of the editor. If I press the arrow up, while in section 2, I move the cursor to the section 1 editor.

![image](https://github.com/sibiraj-s/ngx-editor/assets/82128667/c5ed2c6a-a976-44f3-b685-8608ed662b91)
 you can assume that each input from this picture is an ngx-editor.
